File: src/AndroidNativeBenchmark/app/src/main/java/com/rochapires/admin/androidnativebenchmark/Activities/ActivityListVisualPerformance.java
package com.rochapires.admin.androidnativebenchmark.Activities;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.Menu;
import android.view.MenuItem;
import android.widget.ListView;

import com.rochapires.admin.androidnativebenchmark.R;


public class ActivityListVisualPerformance extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_activity_list_visual_performance);

        ListView listView = (ListView)findViewById(R.id.listView);
        listView.setAdapter(new CustomAdapter(this));
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        // Inflate the menu; this adds items to the action bar if it is present.
        getMenuInflater().inflate(R.menu.menu_activity_visual_performance, menu);
        return true;
    }

}

File: src/AndroidXamarinBenchmark/AndroidXamarinBenchmark/Activities/ActivityListVisualPerformance.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using AndroidXamarinBenchmark.Helpers;

namespace AndroidXamarinBenchmark.Activities
{
    [Activity(Label = "ActivityListVisualPerformance")]
    public class ActivityListVisualPerformance : Activity
    {
        protected override void OnCreate(Bundle bundle)
        {
            base.OnCreate(bundle);

            SetContentView(Resource.Layout.ActivityListVisualPerformance);

            ListView listView = FindViewById<ListView>(Resource.Id.listView);
            listView.Adapter = new CustomAdapter(this);
        }
    }
}